tests/org.jboss.tools.openshift.test/META-INF/MANIFEST.MF

```diff
@@ -27,5 +27,7 @@ Require-Bundle: org.jboss.tools.openshift.client;bundle-version="[3.0.0,4.0.0)",
  org.jboss.tools.common.ui;bundle-version="3.7.1",
  org.eclipse.core.databinding;bundle-version="1.5.0",
  org.eclipse.core.expressions;bundle-version="3.5.0",
- org.apache.commons.io;bundle-version="2.2.0"
+ org.apache.commons.io;bundle-version="2.2.0",
+ org.eclipse.jdt.launching;bundle-version="3.8.0",
+ org.eclipse.jdt.launching.macosx;bundle-version="3.3.0";resolution:=optional

```

Comment:
this caused jenkins to fail
'''
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.jboss.tools.openshift.test 3.2.0.qualifier
[ERROR]   Missing requirement: org.jboss.tools.openshift.test 3.2.0.qualifier requires 'bundle org.eclipse.jdt.launching.macosx 3.3.0' but it could not be found
'''